pvlib/test/test_forecast.py

```diff
@@ -75,6 +75,40 @@ def test_process_data(model):
                               .format(model, variable))
 
 
+@requires_siphon
+def test_bad_kwarg_get_data():
+    amodel = NAM()
+    data = amodel.get_data(_latitude, _longitude, _start, _end,
+                           bad_kwarg=False)
+    assert not data.empty
+
+
+@requires_siphon
+def test_bad_kwarg_get_processed_data():
+    amodel = NAM()
+    data = amodel.get_processed_data(_latitude, _longitude, _start, _end,
+                                     bad_kwarg=False)
+    for variable in _nonnan_variables:
```

Comment:
any particular reason for `assert not data.empty` above but a more detailed check here? I would have guessed that `assert not data.empty` is sufficient here.

Reply:
I copied a different test. I have changed to `assert not data.empty`.